frontend/beCompliant/src/utils/answerCommentMapper.ts

```diff
@@ -0,0 +1,41 @@
+import { Answer, Comment, Table } from '../api/types';
+
+export const addCommentsAndAnswersToRecords = (
+  tableData: Table,
+  commentData: Comment[],
+  answerData: Answer[]
+) => {
+  tableData.records = tableData.records.map((question) => ({
```

Comment:
Hade tenkt at du returnerer verdien her og ikke populerer records  i tillegg. 
```suggestion
  return tableData.records.map((question) => ({
```

Reply:
Anlednignen til det er for at underlette testing og ha bedre struktur. Nå gjører denne funktionen 1 ting mapper, innen gjorde den 2 ting.

Er en bra ting at tenke på at hver hjelpe funktion bare gjører en ting. Da er det enkelt at teste og enkelt at lese